tests/console/valgrind.pm

```diff
@@ -102,7 +102,7 @@ sub run {
     assert_present($output, 'totals: ', "'callgrind totals' mismatch");
 
     # cachegrind tool checks
-    assert_script_run('valgrind --tool=cachegrind --cachegrind-out-file="cachegrind.out" ./valgrind-test');
+    assert_script_run('valgrind --tool=cachegrind --cachegrind-out-file="cachegrind.out" --cache-sim=yes ./valgrind-test');
```

Comment:
is this compatible with all valgrind versions we are running tests with? (think SLE12 SP Maintenance, if there is any valgrind tests)

Reply:
Probably. It's already in a commit from 2009.